sui-execution/v0/sui-adapter/src/programmable_transactions/execution.rs

```diff
@@ -114,7 +114,7 @@ mod checked {
             object_changes,
```

Comment:
Ditto, I think we can revert these changes to v0?

Reply:
not sure what this means..?

Reply:
Looks like you got at it as it has been reverted, thanks!